util/linuxfw/iptables_runner.go

```diff
@@ -437,6 +438,50 @@ func (i *iptablesRunner) DelSNATRule() error {
 	return nil
 }
 
+func buildMagicsockPortRule(port uint16) []string {
+	return []string{"-p", "udp", "--dport", strconv.FormatUint(uint64(port), 10), "-j", "ACCEPT"}
+}
+
+func (i *iptablesRunner) AddMagicsockPortRule(port uint16, network string) error {
```

Comment:
As discussed on VC - Can we add docs to the functions within this PR? While some of them may be quite generic (e.g. x does x, and returns error y in z cases), it can also be a signal to the person who works in this area at a later time that there isn't any 'gotchas' or such. It also makes parsing through functions quicker 😄